src/commands/deploy/deploy.ts

```diff
@@ -63,6 +65,19 @@ async function deploy(actionContext: IActionContext, arg1: vscode.Uri | string |
         validateGlobSettings(context, context.effectiveDeployFsPath);
     }
 
+    if (language === ProjectLanguage.CSharp) {
+        const projFiles: string[] = await dotnetUtils.getProjFiles(language, context.workspaceFolder.uri.fsPath);
+        const mainProject: string = path.join(context.workspaceFolder.uri.fsPath, projFiles[0]);
```

Comment:
I feel like this approach might not work where functions apps are nested inside directory. How do I get access to right `*.csproj` file ?